regression/test.pl

```diff
@@ -115,12 +112,44 @@ ($$$$$)
         last if($included == -1);
         if($result eq "--") {
           $included--;
-        } else {
+        } elsif($result !~ /\s*/) {
```

Comment:
To be consistent with other parts of the code it would be helpful to have a comment to give the English version of what we are doing. like lines 67 and 122.